docs/subscriber/index.rst

```diff
@@ -67,6 +67,17 @@ The method returns a :class:`~.pubsub_v1.types.PullResponse` instance that
 cointains a list of received :class:`~.pubsub_v1.types.ReceivedMessage`
```

Comment:
let's fix "cointains" while we're at it?

Reply:
Sure, why not, will do it.